### DISCUSSION TITLE
Changing Quagga/OSPF MDR

### DISCUSSION THREAD

**Discussion by jhonzy0928** [coreemu/core#747] [General]
**Describe the bug**
First, what I want to do is modify the Quagga routing protocol in the CORE to support multiple nodes using the same routing protocol but with different Hello-interval values in the protocol. So in Figure 1, I add a new class to Quagga's Service.py file and select the routing protocol in CORE. Secondly, as shown in Figure 2, CORE in operation can have connection lines, but I cannot ping any node from the terminal, and there is no node information in the routing neighbor table. Finally, as shown in Figure 3, although there is no information about the neighbor node, I can grab the information exchanged by the beacon (HELLO) using the packet capture tool tcpdump. So is this a BUG? If not, how can I set a different beacon exchange interval for each node? I would appreciate it if you could answer my questions!
**To Reproduce**
1. Modify the file /core/deamon/core/configservices/quagga/service.py，add a new class；
2. Restart core-daemon,create new session,Create a new communication topology；
3. Start，Viewing the routing table，Grab packet；

**Expected behavior**
If it works, each node should be able to set a different HELLO-Interval and work properly
**Screenshots**
![45e9c1ae84b260c21123c511dc86706](https://user-images.githubusercontent.com/54539693/212463554-24b10007-b0fb-4b64-ba0f-ad4887f5bae8.png)
![c019af54d76a548e50652da2333de5c](https://user-images.githubusercontent.com/54539693/212464036-88d489b1-842c-4cac-8839-9160d468ec88.jpg)

![36dc59d9b18135ddc204b42892dee8c](https://user-images.githubusercontent.com/54539693/212463570-376a8069-f575-48c3-ba97-560b0c838545.png)
**Desktop (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04]
 - CORE Version [e.g. 5.2.1]
 - EMANE Version [e.g. 1.3.1]

**Additional context**
Thank you again for reading the full information！

**Comment by bharnden** [comment]
At this point your issue is more with Quagga/OSPF and changing it, than CORE, from what I can see. You may want to use a simple WLAN to avoid EMANE being a factor at all. Then get things running as desired.

CORE will simply run the scripts and configuration you are modifying, after that it has no involvement.
